coordinator/src/main/java/org/apache/uniffle/coordinator/CoordinatorGrpcService.java

```diff
@@ -110,6 +110,7 @@ public void getShuffleAssignments(
     final int replica = request.getDataReplica();
     final Set<String> requiredTags = Sets.newHashSet(request.getRequireTagsList());
     final int requiredShuffleServerNumber = request.getAssignmentShuffleServerNumber();
+    final int estimateTaskConcurrency = request.getEstimateTaskConcurrency();
```

Comment:
What will the value be if the old client request the server?

Reply:
The value will be 0.

Reply:
Will it cause an unexpected result? If not, this feature will be compatible feature.

Reply:
If the estimateTaskConcurrency value is 0, and rss.coordinator.select.partition.strategy=CONTINUOUS, the assignment will be similar to ROUND strategy, you can check CoordinatorUtils#generateRangesGroup.

Reply:
OK.